driver_test.go

```diff
@@ -1793,7 +1793,7 @@ func TestOpenWithTransport(t *testing.T) {
 
 	// Test that transport override also works in insecure mode
```

Comment:
should we change the comment also?

Reply:
Ok, done.

Reply:
it't not changed

Reply:
Actually it is - as you suggested, I left the `insecure mode` part, but added also `OCSP checks disabled`.

Reply:
Finally :)